docs/api/app.md

```diff
@@ -904,6 +904,14 @@ Returns [`ProcessMetric[]`](structures/process-metric.md): Array of `ProcessMetr
 
 Returns [`GPUFeatureStatus`](structures/gpu-feature-status.md) - The Graphics Feature Status from `chrome://gpu/`.
 
+### `app.getGPUInfo(info_type)`
+
+* `info_type` String. Values can be either 'available' for currently available info or `complete` for complete info.
```

Comment:
1. Docs are not consistent with tests: test use `full`/`short` and docs say `complete`/`available`
2. Camel case `info_type` -> `infoType` or `InfoType` ?

Reply:
Yeah, still working on this PR. I have those changes made locally.